backend/dataall/cdkproxy/stacks/cdk_pipeline.py

```diff
@@ -225,11 +225,12 @@ def __init__(
         super().__init__(scope, f"dataall-{{environment_id.title()}}", **kwargs)
         DDKApplicationStack(self, "DataPipeline-{pipeline.label}-{pipeline.DataPipelineUri}", environment_id)
 
+id = f"dataall-cdkpipeline-{pipeline.DataPipelineUri}pip"
```

Comment:
What does pip mean in {pipeline.DataPipelineUri}pip?

Reply:
For the case of CDK-pipelines pipeline type we deploy 2 stacks: one with the repository and one from running CDK deploy on the repo. Their target is the same, except for the pip suffix